lib/modulesync/constants.rb

```diff
@@ -6,5 +6,6 @@ module Constants
     MODULESYNC_CONF_FILE = 'modulesync.yml'
     PROJ_ROOT            = './modules'
     HOOK_FILE            = '.git/hooks/pre-push'
+    GLOBAL_DEFAULTS_KEY  = :global
```

Comment:
having a plain string there avoids colliding with an actual file named 'global'. The yaml would look like this:

```
---
:global:
  key: value
```

Which is e.g. also used by the foreman in its configurations, and I do not think it to be too bad.